types/hellosign-sdk/index.d.ts

```diff
@@ -152,6 +152,7 @@ declare namespace HelloSign {
         details_url: string;
         cc_email_addresses: string | string[];
         signing_redirect_url: string;
+	template_ids: null | string[];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/hellosign-sdk/index.d.ts;
  Line: 155;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Fixed